bundles/org.eclipse.jface.databinding/src/org/eclipse/jface/internal/databinding/viewers/TreeViewerUpdater.java

```diff
@@ -202,7 +204,7 @@ public void remove(Object parent, Object element, int position) {
 			// Only TreeViewer has a remove-by-index method.
 			treeViewer.remove(parent, position);
 		} else {
-			viewer.remove(parent, new Object[] { element });
+			viewer.remove(parent, element);
```

Comment:
@jjohnstn Save action created error in the code here. I really hope I'm not asking you too often - feel free to just ignore if that's the case.